### PR TITLE
[NP-9062] Add WizardFlow support for general admission

### PR DESCRIPTION
**Pull request by KernelDeimos**

### CODE REVIEW

File: src/foam/u2/wizard/crunch/GroupRefinement.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard.crunch',
+  name: 'CapabilityRefinement',
```
Comment (Nauna) [inline src/foam/u2/wizard/crunch/GroupRefinement.js]:
```suggestion
  name: 'GroupRefinement',
```

File: src/foam/u2/wizard/crunch/GroupRefinement.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  refines: 'foam.nanos.crunch.Capability',
```
Comment (Nauna) [inline src/foam/u2/wizard/crunch/GroupRefinement.js]:
```suggestion
  refines: 'foam.nanos.auth.Group',
```

**Review by Nauna — COMMENTED** [review]
overall lgtm

main update here is setting option for
source: group.wizardFlow || group.generalCapability
then plan is a cleanup the rest of the wizards by using dsl exclusively.
Onboard 👍

File: src/foam/u2/crunch/WizardRunner.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+            rootCapability: this.source
+          })
+          .reconfigure('WAOSettingAgent', {
+            waoSetting: foam.u2.crunch.wizardflow.WAOSettingAgent.WAOSetting.UCJ
+          })
+          .remove('RequirementsPreviewAgent')
+      }
+
+      console.error(
+        '%cAre you configuring a new wizard?%c%s',
```
Comment (kgrgreer) [inline src/foam/u2/crunch/WizardRunner.js:108]: I didn't know you could do that.

File: src/foam/nanos/controller/ApplicationController.js [hunk 4]
```diff
@@ -832,44 +834,21 @@ foam.CLASS({
...
-              .addBefore('ConfigureFlowAgent', { class: 'foam.u2.wizard.agents.RootCapabilityAgent', rootCapability: group.generalCapability})
-              .reconfigure('WAOSettingAgent', { waoSetting: foam.u2.crunch.wizardflow.WAOSettingAgent.WAOSetting.UCJ })
-              .remove('RequirementsPreviewAgent')
-              .execute();
-            
-            await this.doGeneralCapabilityPostCheck(ucjCheck);
-          }
 
-        }
+      if ( ! group || ! group.generalCapability ) return true;
```
Comment (sarthak-marwaha) [inline src/foam/nanos/controller/ApplicationController.js:838]:
```suggestion
      if ( ! group?.generalCapability || ! group?.wizardFlow ) return true;
```

Comment (KernelDeimos) [reply src/foam/nanos/controller/ApplicationController.js:838]: we actually need `generalCapability` to be set as well so we know which granted status to check for

File: src/foam/u2/crunch/WizardRunner.js (hunk 3 above)
Comment (KernelDeimos) [reply src/foam/u2/crunch/WizardRunner.js:108]: I did this a bit in the wizard inspector but I only found out recently you can change the font size

**Pull request merged**